man/systemd.exec.xml

```diff
@@ -366,6 +366,22 @@
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BindJournalSockets=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, sockets from <citerefentry>
+        <refentrytitle>systemd-journald.socket</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        will be bind mounted into the mount namespace. This is particularly useful when a different rootfs
+        is used (e.g. through <varname>RootDirectory=</varname> or <varname>RootImage=</varname>), as
+        <varname>MountAPIVFS=yes</varname> mounts an empty <filename>/run/</filename> inside the new root.</para>
+
+        <para>This option is implied when <varname>LogNamespace=</varname> is used,
+        when <varname>MountAPIVFS=yes</varname>, or when <varname>PrivateDevices=yes</varname> is used
```

Comment:
The statements about `MountAPIVFS=` in these two paragraph sounds inconsistent for me...

Reply:
How about "empty /run/" -> "fresh /run/"?